persistence/sql/migrations/sql/20220913091948000000_sessions_devices_add_index.postgres.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX "session_devices_session_id_nid_idx" ON "session_devices" (session_id, nid);
```

Comment:
I think the index down migration (all db versions) should remove the index if possible, or the command should be `CREATE INDEX IF NOT EXIST`. Otherwise it will not be possible to roll back this migration and apply it again

Reply:
Sure. I'll fix it. My thought was that the migration preceding this one is to drop the table considering that it's shipped together

Reply:
Added a migration to drop the index (using IF EXISTS where possible)

Reply:
@aeneasr Scratch that. Just merged the index creation statement with the create table statement. The down migration simply drops the table, removing the need for "drop index" statements.